packages/web/src/pages/upload-page/fields/ReleaseDateField.module.css

```diff
@@ -63,6 +63,15 @@
   border-radius: 4px;
   width: 100%;
   padding: var(--unit-3) var(--unit-4);
+  transition: border ease-in-out 0.1s;
+}
+
+.datePicker:hover {
+  border: 1px solid var(--neutral-light-6);
+}
+
+.datePicker:focus-within {
+  border: 1px solid var(--secondary);
```

Comment:
interesting that both of these colors are outside our named border color variables
something to keep in mind for Harmony colors/borders work